test/readme-version-number-linter.test.js

```diff
@@ -57,6 +57,16 @@ describe('readme-version-number-linter', () => {
       }, tap))
     })
   })
+  describe('readme with out  of date version numbers and no v prefix', () => {
```

Comment:
just because I'm a pain:

```suggestion
  describe('readme with out of date version numbers and no v prefix', () => {
```